sdk/ai/Azure.AI.Projects/tests/AgentClientTests.cs

```diff
@@ -162,14 +162,14 @@ public async Task TestListAgent()
             Assert.AreEqual(0, ids.Count);
             Assert.AreEqual(initialAgentCount + 2, count);
 
-            DeleteAndAssert(client, agent1);
+            await DeleteAndAssert(client, agent1);
             ids.Add(agent1.Id);
             ids.Add(agent2.Id);
             count = await CountElementsAndRemoveIds(client, ids);
             Assert.AreEqual(1, ids.Count);
             Assert.False(ids.Contains(agent2.Id));
             Assert.AreEqual(initialAgentCount + 1, count);
-            DeleteAndAssert(client, agent2);
+            await DeleteAndAssert(client, agent2);
```

Comment:
But we _weren't_ here. I'm betting this last one was failing to wait, the request is finishing _after_ the test teardown functions have run (which means the playback session will no longer be active)